cmd/minikube/cmd/root.go

```diff
@@ -98,6 +99,17 @@ func Execute() {
 		} else {
 			os.Args = append([]string{RootCmd.Use, callingCmd, "--"}, os.Args[1:]...)
 		}
+	} else if filepath.Ext(callingCmd) == ".exe" && driver.IsMicrosoftWSL() {
```

Comment:
instead of checking for the extension of the file (.exe) how about we use runtime.GOOS

Reply:
Yes, the runtime.GOOS shows it is windows even though it is inside WSL. thank you.